lib/core/src/parseCommandLine.cpp

```diff
@@ -418,6 +418,7 @@ parseCmdLineOpt( int argc, char **argv, const char *optString, int includeLong,
             rodsArgs->queryStr = optarg;
             break;
         case 'Q':
+            printf("Warning: RBUDP (-Q) is deprecated. Its use should be avoided.\n");
```

Comment:
Let's remove this since the help text states the option is deprecated.

Reply:
done